build.gradle.kts

```diff
@@ -300,4 +319,64 @@ fun PublicationContainer.setDefaultPomFields() {
       }
     }
   }
+}
+
+fun publishTasks(repository: String): List<Task> {
```

Comment:
This function seems to be unused

Reply:
Not Indeed, thanks! I'll most likely open a new PR with a new branch name as I ended up moving to github actions